pkg/cable/libreswan/libreswan.go

```diff
@@ -45,6 +45,7 @@ import (
 const (
 	cableDriverName = "libreswan"
 	whackTimeout    = 5 * time.Second
+	dpdDelay        = 30
```

Comment:
Please indicate the units:

```suggestion
	dpdDelay        = 30 // seconds
```